specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2023-03-01/PrometheusRuleGroups.json

```diff
@@ -0,0 +1,507 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a PrometheusRuleGroup": {
+            "$ref": "./examples/deletePrometheusRuleGroup.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PrometheusRuleGroupResource": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrometheusRuleGroupProperties",
+          "description": "The Prometheus rule group properties of the resource."
+        }
+      },
+      "description": "The Prometheus rule group resource."
+    },
+    "PrometheusRuleGroupResourcePatchParameters": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "the flag that indicates whether the Prometheus rule group is enabled."
+            }
+          }
+        }
+      },
+      "description": "The Prometheus rule group resource for patch operations."
+    },
+    "PrometheusRuleGroupResourceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrometheusRuleGroupResource"
+          },
+          "description": "the values for the alert rule resources."
+        }
+      },
+      "description": "Represents a collection of alert rule resources."
+    },
+    "PrometheusRuleGroupProperties": {
+      "description": "An Azure Prometheus rule group.",
+      "type": "object",
+      "required": [
+        "scopes",
+        "rules"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Rule group description."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable/disable rule group."
+        },
+        "clusterName": {
+          "type": "string",
+          "description": "Apply rule to data from a specific cluster."
+        },
+        "scopes": {
+          "$ref": "#/definitions/StringArray",
+          "description": "Target Azure Monitor workspaces resource ids. Only one scope currently supported."
```

Comment:
>Only one scope currently supported. [](http://example.com/codeflow?start=72&length=35)

If the intention is to support more scopes in the future, or and/or removing this restriction as a non-breaking change, but to always allow the possibility you'll return multiple scopes in any api-version, maybe you could make it clearer by saying something like 'api-version XX is currently limited to creating one scope. This may change in future .'